pm_mac/pmmacosxcm.c

```diff
@@ -222,6 +228,10 @@ typedef struct coremidi_info_struct {
 /* private function declarations */
 MIDITimeStamp timestamp_pm_to_cm(PmTimestamp timestamp);  // returns host time
 PmTimestamp timestamp_cm_to_pm(MIDITimeStamp timestamp);  // returns ms
+static UInt64 current_host_time(void);
+static UInt64 nanos_to_host_time(UInt64 nanos);
+static UInt64 host_time_to_nanos(UInt64 host_time);
+static UInt64 micros_per_host_tick(void);
 
```

Comment:
portmidi may be overoptimized to use host_time when possible rather than always using nanos (which would give a simpler, smaller set of time functions), but OK, let's extend porttime. These additional functions can exist for apple only (since they are not called from other platforms). Names should be, e.g. Pt_Current_Host_Time -- now I kind of hate this style, but it was originally supposed to be consistent with PortAudio, and better to be consistent that not (or to break everything by changing naming conventions now).

Reply:
> Names should be, e.g. Pt_Current_Host_Time

`Pt_CurrentHostTime`? That would be consistent with the PortMidi naming style:

https://github.com/PortMidi/portmidi/blob/928520f7b79f371854387cb480b4e3b5bf5dac95/pm_common/portmidi.h#L184

Reply:
I've moved these functions to a new header, ptmacosx.h, since they are Apple-specific and so they don't leak into the porttime interface on other platforms:

https://github.com/PortMidi/portmidi/blob/15ddc24129f5e60808026acf6741719277aed916/porttime/ptmacosx.h#L9-L15